app/src/main/java/com/framgia/forder/data/model/DataSuggest.java

```diff
@@ -0,0 +1,67 @@
+package com.framgia.forder.data.model;
+
+import android.os.Parcel;
+import com.arlib.floatingsearchview.suggestions.model.SearchSuggestion;
+import io.realm.RealmObject;
+
+/**
+ * Created by Age on 4/27/2017.
+ */
+
+public class DataSuggest extends RealmObject implements SearchSuggestion {
```

Comment:
ERROR: [ParcelCreator] This class implements `Parcelable` but does not provide a `CREATOR` field
 According to the `Parcelable` interface documentation, "Classes implementing the Parcelable interface must also have a static field called `CREATOR`, which is an object implementing the `Parcelable.Creator` interface."